nodejs/eks/nodegroup.ts

```diff
@@ -565,16 +565,15 @@ async function computeWorkerSubnets(parent: pulumi.Resource, subnetIds: string[]
 /**
  * Iterates through the tags map creating AWS ASG-style tags
  */
-function tagsToAsgTags(tags: {[key: string]: string}): pulumi.Output<string> {
-    let output: pulumi.Input<string> = "";
-    for (const tag of Object.keys(tags)) {
-        output = pulumi.concat(output, pulumi.all([
-            tag,
-            tags[tag],
-        ]).apply(([k, v]) => `
-                          - Key: ${k}
-                            Value: ${v}
-                            PropagateAtLaunch: 'true'`));
-    }
-    return pulumi.output(output);
+function tagsToAsgTags(tagsInput: pulumi.Input<{[key: string]: string}>): pulumi.Output<string> {
+    return pulumi.output(tagsInput).apply(tags => {
```

Comment:
Is `output(tagsInput)` necessary? Pretty sure you can just `apply` from an `Input`

Reply:
`.apply` only [exists](https://pulumi.io/reference/pkg/nodejs/pulumi/pulumi/#OutputInstance-apply) on `Outputs`, so we [convert](https://pulumi.io/reference/programming-model/#frominput) the `Input` to an `Output`.